catalyst_voices/apps/voices/lib/pages/workspace/workspace_error.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_view_models/catalyst_voices_view_models.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+typedef _StateData = ({bool show, LocalizedException? error});
+
+class WorkspaceError extends StatelessWidget {
+  const WorkspaceError({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return _Selector(
+      builder: (context, state) {
```

Comment:
It's difficult to read if you consume the state here but you query it in `_Selector`. To see what the `state` is I need to go to another widget.

Either try to expose what type the state has or just put the BlocSelector here instead of extracting it to another widget

Reply:
Changed all of Workspace widgets so they're private but have `[name]Selector` widgets which is querying bloc